cmd/createorupdate/createorupdate.go

```diff
@@ -21,15 +21,19 @@ import (
 	"github.com/openshift/openshift-azure/pkg/plugin"
 	"github.com/openshift/openshift-azure/pkg/tls"
 	"github.com/openshift/openshift-azure/pkg/upgrade"
+	"github.com/openshift/openshift-azure/pkg/util"
 	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
 )
 
 var logLevel = flag.String("loglevel", "Debug", "valid values are Debug, Info, Warning, Error")
 
 // createOrUpdate simulates the RP
 func createOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCluster, entry *logrus.Entry) (*v20180930preview.OpenShiftManagedCluster, error) {
+
 	// instantiate the plugin
-	p := plugin.NewPlugin(entry, os.Getenv("SYNC_IMAGE"))
+	var config = &acsapi.PluginConfig{SyncImage: os.Getenv("SYNC_IMAGE"),
+		AcceptLanguages: []string{util.LocaleLangToAcceptLanguage(os.Getenv("LANG"))}}
```

Comment:
why not just hard code `en-us` here and drop all of LocaleLangToAcceptLanguage?  Does it really matter to us in our scaffolding implementation?

Reply:
ok, it will make it simpler.